<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1307</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1301</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QPushButton" name="pushButtonNum3">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>190</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNum6">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>280</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNum9">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>370</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditNum">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>100</y>
        <width>841</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNum2">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>190</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNum1">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>190</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNum5">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>280</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNum4">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>280</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNum7">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>370</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNum8">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>370</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNumC">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>460</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNum0">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>460</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNumNosta">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>550</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>VAHVISTA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNumPeruuta">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>550</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PERUUTA</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>210</y>
        <width>291</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>OTA KORTTI</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>180</y>
        <width>291</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>OTA  RAHAT</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QPushButton" name="pushButtonNosta_20">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>371</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>20</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNosta_50">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>190</y>
        <width>371</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string>50</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNosta_90">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>320</y>
        <width>371</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>90</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNosta_120">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>60</y>
        <width>351</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>120</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNosta_200">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>180</y>
        <width>351</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>200</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNostaMuuSumma">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>310</y>
        <width>351</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MUU SUMMA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNostaPeruuta">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>540</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PERUUTA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNostaJaSaldo">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>540</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>NOSTA JA NÄYTÄ SALDO</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNostaRahat">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>540</y>
        <width>281</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>NOSTA</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>10</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>NOSTO</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QPushButton" name="pushButtonNostaJaLahjoita_20">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>351</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>20</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNostaJaLahjoita_50">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>170</y>
        <width>351</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>50</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNostaJaLahjoita_90">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>280</y>
        <width>351</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>90</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNostaJaLahjoita_120">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>60</y>
        <width>351</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>120</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNostaJaLahjoita_200">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>180</y>
        <width>351</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>200</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNostaJaLahjoitaMuuSumma">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>290</y>
        <width>351</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MUU SUMMA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNostaJaLahjoitaPeruuta">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>540</y>
        <width>231</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PERUUTA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNostaJaLahjoitaSaldo">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>540</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>NOSTA JA NÄYTÄ SALDO</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNostaJaLahjoitaRaha">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>540</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>NOSTA</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>10</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>NOSTO</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>420</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>LAHJOITUS:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLahjoitus_1">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>420</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLahjoitus_2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>420</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLahjoitus_5">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>420</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7">
     <widget class="QTextEdit" name="textEditTiliTapahtumat">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>1131</width>
        <height>411</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TILITAPAHTUMAT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonTiliTakaisin">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>520</y>
        <width>351</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TAKAISIN</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowserPvm">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>80</y>
        <width>281</width>
        <height>391</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowserTyyppi">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>80</y>
        <width>541</width>
        <height>391</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>955</x>
        <y>80</y>
        <width>251</width>
        <height>391</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_8">
     <widget class="QPushButton" name="pushButtonSaldoTakasin">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>510</y>
        <width>351</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TAKAISIN</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditSaldo">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>1131</width>
        <height>411</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>20</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SALDO</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="pushButtonNosta">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>351</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>NOSTA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLahjoita">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>210</y>
        <width>351</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>NOSTA JA LAHJOITA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSaldo">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>50</y>
        <width>351</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SALDO</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonTilitapahtumat">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>210</y>
        <width>351</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TILITAPAHTUMAT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonUlos">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>520</y>
        <width>351</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>KIRJAUDU ULOS</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>230</y>
        <width>311</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SYÖTÄ KORTTI</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_9">
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>150</y>
        <width>421</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>OLET KIRJAUTUNUT ULOS</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>400</y>
        <width>211</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>OTA KORTTI</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1307</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
